ehrapy/preprocessing/_scanpy_pp_api.py

```diff
@@ -128,20 +131,74 @@ def subsample(
     random_state: AnyRandom = 0,
     copy: bool = False,
 ) -> AnnData | None:  # pragma: no cover
-    """Subsample to a fraction of the number of observations.
+    warnings.warn(
+        "This function is deprecated and will be removed in the next release. Use ep.pp.sample instead.",
+        DeprecationWarning,
+        stacklevel=2,
+    )
+    return sample(data=data, fraction=fraction, n_obs=n_obs, rng=random_state, copy=copy)
+
+
+def sample(
+    data: AnnData | np.ndarray,
+    fraction: float | None = None,
+    *,
+    n_obs: int | None = None,
+    rng: int | None = None,
+    balanced: bool = False,
+    method: Literal["under", "over"] = "under",
+    key: str | None = None,
+    copy: bool = False,
+    replace: bool = False,
+    axis: Literal["obs", 0, "var", 1] = "obs",
+    p: str | NDArray[np.bool_] | NDArray[np.floating] | None = None,
+) -> AnnData | None | tuple[np.ndarray, np.ndarray]:  # pragma: no cover
+    """Sample a fraction or a number of observations / variables with or without replacement.
 
     Args:
         data: The (annotated) data matrix of shape `n_obs` × `n_vars`. Rows correspond to observations (patients) and columns to features.
-        fraction: Subsample to this `fraction` of the number of observations.
-        n_obs: Subsample to this number of observations.
-        random_state: Random seed to change subsampling.
+        fraction: Sample to this `fraction` of the number of observations.
+        n_obs: Sample to this number of observations.
+        rng: Random seed to change subsampling.
```

Comment:
rng: Random seed to change subsampling.
```suggestion
        rng: Random seed.
```